test/viewmodels/application/AgentTraderDetailsSummarySpec.scala

```diff
@@ -63,7 +64,8 @@ class AgentTraderDetailsSummarySpec extends AnyFreeSpec with Matchers {
     None,
     "NE1 4FF",
     "country code",
-    None
+    None,
+    Some(false)
```

Comment:
Should this be None if the value is minimum trader details?

Reply:
A trader always has a value (unless it's an old application). It's always set to either Some(true) or Some(false).

I don't think it really matters for this test though